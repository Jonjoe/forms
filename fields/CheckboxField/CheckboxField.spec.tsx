import React from "react";
import renderer from "react-test-renderer";

import Example from "./CheckboxField.example";

test("CheckboxField renders correctly", () => {
  const component = renderer.create(<Example />);

  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
