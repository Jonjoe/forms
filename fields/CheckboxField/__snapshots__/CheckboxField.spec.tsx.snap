// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CheckboxField renders correctly 1`] = `
<input
  className="CheckboxField "
  id="example"
  onClick={[Function]}
  type="checkbox"
/>
`;
